### PR TITLE
Platform: Review consistency of READMEs in repos [#97090508]

### PR DESCRIPTION
**Pull request by patientfrenzy**
Update README to ensure it has setup, test and deploy instructions.

- Indicate user can setup app using specific ht laptop command
- Detail setup for deploy, specify staging and production deploy instructions

===

Pivotal tracker story [#97090508](https://www.pivotaltracker.com/story/show/97090508) in project *Platform*:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,60 +66,25 @@ A subscriber can "catch up" event if it hasn't pulled events for a while
...
+### Setup
 
-```
-gem install tunnels
-sudo tunnels 127.0.0.1:443 127.0.0.1:80
-```
+####Installation
 
-This command creates a tunnel between port 443 (the default SSL port) and your 80 port.
+**NOTE: The ht-laptop command is by far the best way to set your machine up, it will get the web app running for you plus all other dependencies.**
```
Comment (pedrocunha) [inline README.md]: `routemaster` is a public project. `ht-laptop` is not open-sourced yet :) we might want to keep this documentation private for the time being? What do you think ?

File: README.md [hunk 2]
```diff
@@ -152,6 +111,69 @@ This will start both the **web** and **watch** processes. Keep in mind that the
...
+
+To run the test suite:
+
+```
+bundle exec rspec
+```
+
+There is currently no travis project for Routemaster
+
+###Deploy
```
Comment (pedrocunha) [inline README.md]: I think we might want to keep this away from the README, others people setup might differ from ours. What do you think ?

**Comment by pedrocunha** [comment]
Thanks for refactoring this, I wonder if we are introducing too much HouseTrip code behaviour in this opensourced project.

**Comment by patientfrenzy** [comment]
@pedrocunha I agree - originally I hadn't realised it was open sourced, will amend

File: README.md (hunk 2 above)
Comment (patientfrenzy) [inline README.md]: agreed

**Pull request closed without merging**